actions/llm-action-error-reporter/action.yml

```diff
@@ -142,56 +192,134 @@ runs:
       env:
         GH_TOKEN: ${{ inputs.gh-token }}
         PR_NUMBER: ${{ steps.get_pr_number.outputs.pr_number }}
-        edit_comment: ${{ inputs.edit-comment }}
       run: |
         workflow_name=$(gh run view ${{ github.event.workflow_run.id }} --json workflowName --jq '.workflowName')
         head_sha=$(gh run view ${{ github.event.workflow_run.id }} --json headSha --jq '.headSha')
-        short_sha=$(echo $head_sha | cut -c1-7)
         repo_url=$(gh repo view --json url --jq '.url')
+        wf_job_urls=$(gh run view ${{ github.event.workflow_run.id }} --json jobs --jq '.jobs | map("[\(.name)](\(.url))") | join(" , ")')
+        workflow_id=$(echo "$workflow_name" | tr -d ' ' | tr '[:upper:]' '[:lower:]')
 
-        pr_message="**Below is an analysis created by an LLM. Be mindful of hallucinations and verify accuracy.**
+        pr_message="
+        <$workflow_id>
 
-        ## WF: $workflow_name[#$short_sha]($repo_url/commit/$head_sha)
+        ## AER Report: [$workflow_name]($repo_url/actions/runs/${{ github.event.workflow_run.id }})
```

Comment:
Want to link to the specific job here? This link goes the run that called this workflow (the originating workflow).

Wondering if we should change this to use: `https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}#${GITHUB_JOB})`

This should go to the exact run and job than ran this this `llm-action-error-report` action.

Reply:
@chainchad 
Thank you! Made some changes and resolved some DRY issues.

So this is how I imagined the script would work:
- if someone sees the error report then they may want to confirm the source
  - this will allow them to do so without scroll to the bottom or go to the actions tab
- links for specific jobs that errored out are provided as well to reduce a few more clicks
- now, the link to the AER workflow itself is probably not useful here
  - it wouldn't give out any information, just that OpenAI request is successful
  - and if the AER workflow fails then the comment wouldn't happen
  - but I've also added the link under the title just in case
  
Also noticed that `${GITHUB_RUN_ID}#${GITHUB_JOB}` url wouldn't work, as `$GITHUB_JOB` just returns the job name, and the URL gets redirected to the workflow run page directly. Directly linking to specific jobs seems to need `[run_id]/jobs/[job_number]`

Updated the example for the [test pr as well](https://github.com/smartcontractkit/aer_tester/pull/4).